draft-wang-ppm-differential-privacy.md

```diff
@@ -193,13 +205,18 @@ values they produce for the Collector).
 In this document, we shall refer to the composition of DP mechanisms into a
 scheme that provides (some notion of) DP as a "DP policy". For some policies,
 noise is added only by the Clients or only by the Aggregators, but for others,
-noise might be added by both Clients and Aggregators.
+noise might be added by both Clients and Aggregators. The best choice of policy
+depends one's "trust model": if most Clients can be trusted to noise their
+measurements correctly, then a high degree of privacy can be achieved with
+minimal loss in utility; but if the trust model includes the possibility that many Clients are corrupted by the attacker,
+then the same degree of privacy can only be achieved at the cost of higher
```

Comment:
nit: "at the cost of some utility" because "higher" is weird here since the utility is getting lower.

Reply:
Good catch, I meant "decreased" 😬